src/parquet/thrift/util.h

```diff
@@ -34,13 +35,44 @@ inline void DeserializeThriftMsg(const uint8_t* buf, uint32_t* len, T* deseriali
       tproto_factory.getProtocol(tmem_transport);
   try {
     deserialized_msg->read(tproto.get());
-  } catch (apache::thrift::protocol::TProtocolException& e) {
-    throw ParquetException("Couldn't deserialize thrift.", e);
+  } catch (std::exception& e) {
+    //TODO Determine an appproach to log exceptions
+    LOG(1) << "Couldn't deserialize thrift: " << e.what();
+    return false;
```

Comment:
I will append all the exceptions inside `serialized-page.cc` and throw in this case.